media-tv/mythtv/mythtv-31.0.ebuild

```diff
@@ -0,0 +1,435 @@
+		$(use_enable hdpvr)
+		$(use_enable dvb)
+		$(use_enable asi)
+	)
+
+	# Video Output Support
+	myconf+=(
+		$(use_enable X x11)
+		$(use_enable X xrandr)
+	)
+
+	# Hardware accellerators
+	myconf+=(
+		$(use_enable nvdec)
+		$(use_enable vaapi)
+		$(use_enable vdpau)
+		$(use_enable opengl)
+		$(use_enable libass)
+	)
+
+	# System tools
+	myconf+=(
+		$(use_enable systemd systemd_notify)
+		$(use_enable systemd systemd_journal)
+		$(use_enable xml libxml2)
+		$(use_enable zeroconf libdns-sd)
+	)
+
+	# Bindings
+	if use perl && use python; then
+		myconf+=(--with-bindings=perl,python)
+	elif use perl; then
+		myconf+=(--without-bindings=python)
+		myconf+=(--with-bindings=perl)
+	elif use python; then
+		myconf+=(--without-bindings=perl)
+		myconf+=(--with-bindings=python)
+	else
+		myconf+=(--without-bindings=perl,python)
+	fi
+	use python && myconf+=(--python="${EPYTHON}")
+	myconf+=($(use_enable java bdjava))
+
+	# External codec library options (used for mythffmpeg and streaming transcode)
+	# lame is required for some broadcasts for silence detection of commercials
+	# default enable in IUSE with +lame
+	myconf+=(
+		$(use_enable lame libmp3lame)
+		$(use_enable xvid libxvid)
+		$(use_enable x264 libx264)
+		$(use_enable x265 libx265)
+		$(use_enable vpx libvpx)
+	)
+
+	# Clean up DSO load times and other compiler bits
+	myconf+=(--enable-symbol-visibility)
+	myconf+=(--enable-pic)
+
+	if tc-is-cross-compiler ; then
+		myconf+=(--enable-cross-compile --arch=$(tc-arch-kernel))
+		myconf+=(--cross-prefix="${CHOST}"-)
+	fi
+
+	myconf+=($(use_enable bluray libbluray_external))
+
+	# econf sets these options that are not handled by configure:
+	# --build --host --infodir --localstatedir --sysconfdir
+
+	einfo "Running ./configure ${myconf[@]} - THIS MAY TAKE A WHILE."
+	./configure \
+		--prefix="${EPREFIX}/usr" \
+		--cc="$(tc-getCC)" \
+		--cxx="$(tc-getCXX)" \
+		--ar="$(tc-getAR)" \
+		--optflags="${CFLAGS}" \
+		--extra-cflags="${CFLAGS}" \
+		--extra-cxxflags="${CXXFLAGS}" \
+		--extra-ldflags="${LDFLAGS}" \
+		--qmake=$(qt5_get_bindir)/qmake \
+		"${myconf[@]}"
+}
+
+src_install() {
+	emake STRIP="true" INSTALL_ROOT="${D}" install
+	use python && python_optimize  # does all packages by default
+	dodoc AUTHORS UPGRADING README
+	readme.gentoo_create_doc
+
+	insinto /usr/share/mythtv/database
+	doins database/*
+
+	newinitd "${FILESDIR}"/mythbackend.init-r3 mythbackend
+	newconfd "${FILESDIR}"/mythbackend.conf-r1 mythbackend
+	if use systemd; then
+		systemd_newunit "${FILESDIR}"/mythbackend.service-28 mythbackend.service
+	fi
+
+	dodoc keys.txt
+
+	keepdir /etc/mythtv
+	fowners -R mythtv /etc/mythtv
+	keepdir /var/log/mythtv
+	fowners -R mythtv /var/log/mythtv
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/mythtv.logrotate.d-r4 mythtv
+
+	insinto /usr/share/mythtv/contrib
+	# Ensure we don't install scripts needing the perl bindings (bug #516968) Finding none is OK.
+	if use perl; then
+		find contrib/ -name '*.pl' -exec rm {} \;
```

Comment:
The `|| die` is also if `find` executable cannot be found or something. You need to be aware when this files exists and when not, but I believe the USE flag is used for that.

Reply:
The files being removed are user contributed scripts that are sometimes helpful when someone screws up their system. They are never required for the operation of MythTV. Installing the system should not fail even it "find" is not available. The third "find" does fail if scripts that remain can not be made executable if enabled by USE flag.